development/build/scripts.js

```diff
@@ -205,171 +216,250 @@ function createScriptTasks({ browserPlatforms, livereload }) {
           : externalDependenciesMap[contentscript],
         devMode,
         testing,
+        browserPlatforms,
       }),
     );
   }
+}
 
-  function bundleTask(opts) {
-    let bundler;
-
-    return performBundle;
-
-    async function performBundle() {
-      // initialize bundler if not available yet
-      // dont create bundler until task is actually run
-      if (!bundler) {
-        bundler = generateBundler(opts, performBundle);
-        // output build logs to terminal
-        bundler.on('log', log);
-      }
+function createNormalBundle({
+  filename,
+  filepath,
+  extraEntries = [],
+  modulesToExpose,
+  externalDependencies,
+  devMode,
+  testing,
+  browserPlatforms,
+}) {
+  return async function () {
+    // create bundler setup and apply defaults
+    const buildConfiguration = createBuildConfiguration();
+    const { bundlerOpts } = buildConfiguration;
+
+    const envVars = getEnvironmentVariables({ devMode, testing });
+    setupBundlerDefaults(buildConfiguration, {
+      filename,
+      devMode,
+      envVars,
+      browserPlatforms,
+    });
 
-      const buildPipeline = [
-        bundler.bundle(),
-        // convert bundle stream to gulp vinyl stream
-        source(opts.filename),
-        // Initialize Source Maps
-        buffer(),
-        // loads map from browserify file
-        sourcemaps.init({ loadMaps: true }),
-      ];
-
-      // Minification
-      if (!opts.devMode) {
-        buildPipeline.push(
-          terser({
-            mangle: {
-              reserved: ['MetamaskInpageProvider'],
-            },
-            sourceMap: {
-              content: true,
-            },
-          }),
-        );
-      }
+    // set bundle entry file
+    bundlerOpts.entries = [...extraEntries, filepath];
```

Comment:
`filepath` is `undefined` for the deps bundles. Previously there was a `buildLib` flag that dictated whether `entries` was set. It should be left unset when building the deps bundles.